packages/ampath-esm-clinic-dashboard-hiv-lists/src/custom-hooks/useCurrentLocation.tsx

```diff
@@ -0,0 +1,5 @@
+const locationUuidKey = "location_uuid";
+
+export function useCurrentLocation() {

```

Comment:
We could annotate this to `useCurrentLocation(): string | null`. Maybe worth adding in a default value as well? Or if not, at least asserting that it's not null at the point of use.